.github/policies/resourceManagement.yml

```diff
@@ -516,5 +516,14 @@ configuration:
       - removeLabel:
           label: Status:No recent activity
       description: '[stale community PR] [5-4] Remove "Status:No recent activity" if there are any reviews.'
+    - if:
+      - payloadType: Issues
+      - labelAdded:
+          label: breaking-change
+      then:
+      - addReply:
+          reply: >-
+            Refer to the [.NET SDK breaking change guidelines](https://github.com/dotnet/sdk/blob/main/documentation/project-docs/breaking-change-guidelines.md#required-process-for-all-net-sdk-breaking-changes)

```

Comment:
Maybe put an "if" or something to qualify that this is only needed when an issue is .NET SDK based?